Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/DialogHelper.kt

```diff
@@ -17,9 +16,10 @@ object DialogHelper {
     )
 
     fun showDialog(
+        activity: Activity,
```

Comment:
whats the motivation for adding this here?

Reply:
Dialogs need an activity as bounded context.

Reply:
sorry, this was referring to moving the activity from the DialogInstance to the showDialog method

Reply:
since you can instantiate a DialogHelper at any time but the activity that you used to instantiate it might go stale and result in a crash when trying to show the dialog.
I thought it would be safer to move the activity declaration closer to where it is actually used